packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -21,6 +20,7 @@ export interface GeneratorContext {
   zipUrl?: string;
   zip?: AdmZip;
   fallbackZipPath?: string;
+  cancelDownloading?: boolean;
```

Comment:
Is there any problem if don't add this `cancelDownloading` property to context?

Reply:
I think context is used for the communication between actions. We need to inform the action `fetchZipFromUrlAction` that the downloading process is canceled. Otherwise, it finds the zipUrl is not fulfilled and throw an unexpected error.